lms/templates/help_modal.html

```diff
@@ -149,11 +158,106 @@
+            if (userAuthenticated && finishedLoadingCourseOptions && courseOptions.length > 1) {
+                $('#feedback_form_course').html(courseOptions.join(''));
+                $('.js_feedback_form_course_select').removeClass('hidden');
+            }
+            $closeButton.focus();
+        },
+        loadCourseOptions = function() {
+            // Prevent race condition that can result from multiple requests being made at the same time.
+            if (courseOptionsLoadInProgress) {
+                return;
+            }
+
+            courseOptionsLoadInProgress = true;
+            $.ajax({
+                url: '/api/enrollment/v1/enrollment',
+                success: function(data) {
+                    var i,
+                        courseDetails,
+                        courseName,
+                        courseId,
+                        option,
+                        defaultOptionText = '${_("- Select -") | n, js_escaped_string}',
+                        markedSelectedOption = false;
+
+                    // Make sure courseOptions is empty before we begin pushing options into it.
+                    courseOptions = [];
+
+                    for (i = 0; i < data.length; i++) {
+                        courseDetails = data[i]['course_details'];
+                        if (!courseDetails) {
+                            continue;
+                        }
+
+                        courseName = courseDetails['course_name'];
+                        courseId = courseDetails['course_id'];
+                        if (!(courseName && courseId)) {
+                            continue;
+                        }
+
+                        // Build an option for this course and select it if it's the course we're currently viewing.
+                        if (!markedSelectedOption && courseId === currentCourseId) {
+                            option = buildCourseOption(courseName, courseId, true);
+                            markedSelectedOption = true;
+                        } else {
+                            option = buildCourseOption(courseName, courseId, false);
+                        }
+
+                        courseOptions.push(option);
+                    }
+
+                    // Build the default option and select it if we haven't already selected another option.
+                    if (markedSelectedOption) {
+                        option = buildCourseOption(defaultOptionText, '', false);
+                    } else {
+                        option = buildCourseOption(defaultOptionText, '', true);
+                    }
+
+                    // Add the default select option to the head of the courseOptions Array.
+                    courseOptions.unshift(option);
+
+                    finishedLoadingCourseOptions = true;
+                },
+                complete: function() {
+                    courseOptionsLoadInProgress = false;
```

Comment:
Reset this after the request finishes so that if something went wrong, the request can be repeated the next time the user opens the help tab.